src/i18n/en.json

```diff
@@ -0,0 +1,138 @@
+{
+  "common": {
+    "cancel": "Cancel",
+    "copyright_MD": "© {year} Doppler LLC. All rights reserved. [Privacy and Legal Policy](https://www.fromdoppler.com/en/legal/privacy-policy).",
+    "help": "Help",
+    "hide": "Hide",
+    "message": "Message",
+    "recaptcha_legal_HTML": "Site protected by reCAPTCHA and the Google <a href=\"https://policies.google.com/privacy?hl=en\">Privacy Policy</a> and <a href=\"https://policies.google.com/terms?hl=en\">Terms of Service</a>.",
+    "send": "Send",
+    "show": "Show"
+  },
+  "feature_panel": {
+    "email_automation": "Email automation",
```

Comment:
`Email Automation`?

Reply:
@carodipietro esto que comenta @cbernat no es parte de los cambios de @GusBaamonde, pero me parece que los comentarios de Cecilia son correctos, ¿Crees que deberíamos aplicar esta sugerencia?

Reply:
It should be `Email Automation`